policyengine_uk/parameters/gov/dfe/extended_childcare_entitlement/expense_rate.yaml

```diff
@@ -0,0 +1,28 @@
+description: The Department for Education sets hourly rates to convert childcare hours into expenses for extended childcare entitlement. These rates vary by child's age and are used to calculate the funding provided to childcare providers.
+
+metadata:
+  type: single_amount
+  threshold_unit: year
+  amount_unit: currency-GBP
+  label: Extended childcare entitlement hourly funding rates by age
```

Comment:
```suggestion
  label: Extended childcare entitlement hourly rates
```

Reply:
Do the US devs adopt this pattern? I'm not sure we have discussed it here.